.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need this file in our repo if this file is "editor" file ? If all of us were using Webstorm IDE then... let it be, but I'm not using it, so I don't need them.